modules/admin_manual/pages/configuration/server/occ_commands/app_commands/richdocuments.adoc

```diff
@@ -0,0 +1,84 @@
+= Collabora Online / Secure View
+
+Marketplace URL: {oc-marketplace-url}/apps/richdocuments[Collabora Online]
+
+Collabora Online allows you to to work with all kinds of collabora office documents inside ownCloud for improved productivity. For details please see the
+xref:enterprise/collaboration/collabora_secure_view.adoc[Collabora Online / Secure View] documentation.
+
+`config:app` commands configure the Collabora Online app.
+
+[source,console]
+----
+ config:app:get|set
+ richdocuments wopi_url            Number of days passed after upload (or restore)
```

Comment:
```suggestion
 richdocuments wopi_url            ...
```

Reply:
If you do that, it will render the following:
```
richdocuments wopi_url            ...
                                  that files will be archived
```
Are you sure?